src/edgepi/pwm/README.md

```diff
@@ -14,16 +14,17 @@ edgepi_pwm = EdgePiPWM()
 
 # Initialize and open pwm device
 edgepi_pwm.init_pwm(PWMPins.PWM1)
-# Setting PWM parameters
-edgepi_pwm.set_config(PWMPins.PWM1, 1000.0, 50.0)
-# enable the pwm signal 
+# Setting PWM parameters, frequency 1K to 10K, duty cycle form 0-1.0
+edgepi_pwm.set_config(PWMPins.PWM1, 1000.0, 0.50)
+# enable the pwm signal, PWM duty cycle and frequency must be set to enable the singal
 edgepi_pwm.enable()
 # disable the pwm signal 
 edgepi_pwm.disable()
 # Close PWM device
 edgepi_pwm.close()
 
 ```
+__NOTE__: Frequency and duty-cycle must be set before enabling the PWM signal.
```

Comment:
I think you only need to set frequency.

Reply:
I think it is good idea to enforce them to set dutycycle at the same time, so they don't have to call the function twice. And it doesn't make sense to only set frequency

Reply:
[a4aea98](https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/369/commits/a4aea98c56ec92b8ec443c416225e247803b332f)